generate_training_data_scripts/README.rst

```diff
@@ -0,0 +1,75 @@
+..
+    This file is part of INSPIRE.
+    Copyright (C) 2014-2018 CERN.
+
+    INSPIRE is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    INSPIRE is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+
+    In applying this license, CERN does not waive the privileges and immunities
+    granted to it by virtue of its status as an Intergovernmental Organization
+    or submit itself to any jurisdiction.
+
+==========================
+Generate the training data
+==========================
+
+The training data can be generated, but in a somewhat complex way. Data for Core and Non-Core records needs to be generated separately from data for the Rejected records.
+
+Generate Core and Non-Core data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Core and Non-Core records can be extracted from the INSPIRE dump data migrated on a local inspire-next instance. First, we would need to setup `inspire-next <https://github.com/inspirehep/inspire-next>`_ locally. Then, the code snippet specified in ``generate_core_and_noncore_data.py`` is to be run from within the inspirehep shell.
+
+This produces two files: *inspire_core_records.json* and *inspire_noncore_records.json*. We will later combine this data with the Rejected records data.
+
+Here, we consider only data from 2016 onwards since before that the curation rules for classification as Core, Non-Core, and Rejected were different. However, the user is free to modify the earliest_date field in the script to specify a starting data of their choice.
+
+Generate Rejected data
+^^^^^^^^^^^^^^^^^^^^^^
+
+The data for Rejected articles is extracted from in a hackish way. The workflows themselves need to be modified in our local inspire-next setup. First, the file *inspire-next/inspirehep/modules/workflows/workflows/article.py* needs to be modified as specified in ``article.py``. We need to add another file *inspire-next/inspirehep/modules/workflows/tasks/makejson.py* with the contents of ``makejson.py``.
```

Comment:
The data for Rejected articles is extracted **from** in a hackish way.

I feel like that ``from`` just slipped in the sentence and should be removed?